sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientAsyncTest.java

```diff
@@ -38,100 +38,77 @@ static void afterAll() {
     }
 
     private ContainerRegistryAsyncClient getContainerRegistryAsyncClient(HttpClient httpClient) {
-        return getContainerRegistryBuilder(httpClient).buildContainerRegistryAsyncClient();
-    }
-
-    private void validateListResponse(PagedResponse<String> pagedResponse) {
-        assertNotNull(pagedResponse);
-        List<String> repositories = pagedResponse.getValue();
-        assertEquals(3, repositories.size());
-        assertEquals(repositories.get(0), "dockercloud/hello-world");
-        assertEquals(repositories.get(1), "library/hello-seattle");
-        assertEquals(repositories.get(2), "library/hello-world");
+        return getContainerRegistryBuilder(httpClient).buildAsyncClient();
     }
 
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("getHttpClients")
     public void listRepositories(HttpClient httpClient) {
```

Comment:
If possible, we should expose a way to test both sync and async clients and xxx and xxxWithResponse methods with paramerized annotation since they are hand authored today.